.ci/install-azure-cli.sh

```diff
@@ -5,4 +5,4 @@ set -e -o pipefail
 apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
```

Comment:
Replace this with `curl -L https://packages.microsoft.com/keys/microsoft.asc | apt-key add -`